Installer/dolphin-emu-nogui.6

```diff
@@ -0,0 +1,41 @@
+.TH DOLPHIN-EMU-NOGUI 6 "March 3, 2016"
+.SH NAME
+\fBdolphin-emu-nogui\fR is an emulator for running GameCube, Wii, and Triforce games on
+Windows, Linux, OS X, and recent Android devices.
+.SH SYNOPSIS
+.B dolphin-emu-nogui
+[\fB-h\fR] [\fB-d\fR] [\fB-l\fR] [\fB-e \fIROM\fR] [\fB-b\fR] [\fB-V \fIVIDEO\fR] [\fB-A \fIAUDIO\fR]
+.SH DESCRIPTION
+Dolphin is a Gamecube, Wii and Triforce (the arcade machine based on the
+Gamecube) emulator, which supports full HD video with several enhancements such
+as compatibility with all PC controllers, turbo speed, networked multiplayer,
+and more.
```

Comment:
The "C" in GameCube should be capitalized. Also where is this description from? Perhaps the one from the Dolphin homepage should be used instead.